pyproject.toml

```
@@ -0,0 +1,37 @@
+[project]
+name = "matplotlib"
+description = "Python plotting package"
+readme = "README.rst"
+license = { file = "LICENSE/LICENSE" }
+dynamic = ["authors", "dependencies", "requires-python", "version"]
+classifiers=[
+    'Development Status :: 5 - Production/Stable',
+    'Framework :: Matplotlib',
+    'Intended Audience :: Science/Research',
+    'Intended Audience :: Education',
+    'License :: OSI Approved :: Python Software Foundation License',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Topic :: Scientific/Engineering :: Visualization',
+]
+
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=7", "wheel", "certifi>=2020.06.20", "numpy>=1.19"]
```

This should be `oldest-supported-numpy`, see https://github.com/scipy/oldest-supported-numpy

The problem is that if you put a floor here, then when the background env gets made it will pull the newest version of numpy it can find.  Numpy promises that things complied with old numpy will work with new numpy at runmtime, but they do not promise that things compiled with new numpy will work with old numpy at runtime.

```suggestion
requires = ["setuptools>=45", "setuptools_scm[toml]>=7", "wheel", "certifi>=2020.06.20", "oldest-supported-numpy"]
```

---

Now it seems like the following happens:
```
  xarray 2022.6.0 requires numpy>=1.19, but you have numpy 1.17.3 which is incompatible.
  pandas 1.4.4 requires numpy>=1.18.5; platform_machine != "aarch64" and platform_machine != "arm64" and python_version < "3.10", but you have numpy 1.17.3 which is incompatible.
```